front-end/src/app/app.component.ts

```diff
@@ -1,10 +1,43 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
+class Task {
+  title = '';
+  comment = '';
+
+  constructor(titl: string, comm: string) {
+    this.title = titl;
+    this.comment = comm;
+  }
+}
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-  title = 'to_do';
+  taskArray: Task[] = [];
+  doneArray: Task[] = [];
+  renderClearbtn = false;
+
+  addTask(title: string, comment: string) {
+    if (title != '') {
+      var temp = new Task(title, comment);
+      this.taskArray.push(temp);
+    } else alert("Task can't be empty!");
+  }
+
+  taskDone(i: number) {
+    this.doneArray.push(this.taskArray[i]);
+    this.taskArray.splice(i, 1);
```

Comment:
a `taskDelete` függvény is ugyan ezt csinálja, ergo kód duplikációt csináltál

Mi lenne akkor, ha módosítani kellene a törlésen? (pl. ha majd backend-et kell hívni a jövőben, nem pedig simán splice-olni) -> Ilyen esetben majd minden helyen át kell írnod a kódot, ahol duplikáció van. 

Ennél egyszerűbb megoldás, ha itt simán meghívod `taskDelete` függvényt. Ha majd abban a függvényben átírod a törléskor lefutó kódot, akkor automatikusan ez a `taskDone` függvény is az elvárt módon fog működni